lua/ulx/modules/sh/cfc_cleanup.lua

```diff
@@ -34,21 +34,26 @@ ropes:defaultAccess( ULib.ACCESS_ADMIN )
 ropes:help( "Remove target(s) ropes" )
 
 
-local function cleanupPlayerEnts( callingPlayer, targetPlayer )
+local function cleanupPlayerEnts( callingPlayer, targetPlayers )
+    local isTarget = {}
+    for _, ply in pairs( targetPlayers ) do
+        isTarget[ply] = true
+    end
+    
     local count = 0
     for _, ent in ipairs( ents.GetAll() ) do
         local owner = ent.CPPIGetOwner and ent:CPPIGetOwner()
-        if owner == targetPlayer and not ent:IsWeapon() then
+        if isTarget[owner] and not ent:IsWeapon() then
             ent:Remove()
             count = count + 1
         end
     end
 
-    ulx.fancyLogAdmin( callingPlayer,  "#A removed "..count.." entities owned by #T", targetPlayer )
+    ulx.fancyLogAdmin( callingPlayer,  "#A removed "..count.." entities owned by #T", targetPlayers )
```

Comment:
May want to split up the counts to print directly to the admin, too. How will this work? What will a table of players look like?

Reply:
ulx formats it something like this
`HMM removed 1000 entities owned by Phatso, HMM, and Bleck`
`HMM removed 1000 entities owned by Everyone`

Reply:
Could probably copy/do the same as i did with freeze command